src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/resource/VideoLinkBookingController.kt

```diff
@@ -292,4 +293,50 @@ class VideoLinkBookingController(
     request: RequestVideoBookingRequest,
     httpRequest: HttpServletRequest,
   ) = requestBookingService.request(request, httpRequest.getBvlsRequestContext().user)
+
+  @Operation(summary = "Endpoint to search for a unique matching video link booking.")
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "200",
+        description = "The matching video link booking details",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = VideoLinkBooking::class),
```

Comment:
So I am using the same model object as the one used by find by id.  Really we are not supposed to reuse model objects.  This is being used in the same controller so I felt this was acceptable in this situation.

Happy to change if peeps feel strongly about it.